tests/test_service.py

```diff
@@ -146,7 +146,48 @@ def test_dwd_readings_no_period():
 
 
 @pytest.mark.sql
-def test_dwd_readings_sql(dicts_are_same):
+def test_dwd_readings_sql_tabular(dicts_are_same):
+
+    response = client.get(
+        "/api/dwd/observations/values",
+        params={
+            "tabular": True,
+            "station": "01048,4411",
+            "parameter": "kl",
+            "resolution": "daily",
+            "period": "recent",
+            "sql": "SELECT * FROM data WHERE temperature_air_max_200 < 2.0",
```

Comment:
This test is flaky as recent is limited to 500 days for daily data. Better take historical and apply sql on that.

Reply:
Thank you. I just added the datetime constraint `"date": "2019/2022"`.

Reply:
Thanks! However I think this won't work as I had changed the API to take data only from given periods if both a date range and period are given!

Reply:
Thank you. I tried two things:
- Remove `period` parameter.
- Adjust to `"period": "recent,historical"`.

On both occasions, the test croaks. I believe we will have to rework lots of tests to regain stability as soon as the 2019 data will be switched out of the "recent" period, right? I would like to address this within a separate patch.

Reply:
> we will have to rework lots of tests

Also, I found that `"date": "2019/2022"` does not DWIM right now. It will expand to 2019-01-01/2022-01-01, however, by adding more smartness, I would like it to expand to 2019-01-01/2022-12-31, like the [`filter_by_date()`](https://github.com/earthobservations/wetterdienst/blob/0.15.0/wetterdienst/dwd/pandas.py#L54) implementation does.

I would also like to address that on behalf of a separate patch.

Reply:
I've added #385 in order to track those remarks.